var/spack/repos/builtin/packages/py-pennylane-lightning/package.py

```diff
@@ -7,20 +7,123 @@
 from spack.package import *
 
 
-class PyPennylaneLightning(PythonPackage):
-    """PennyLane-Lightning plugin"""
+class PyPennylaneLightning(CMakePackage, PythonExtension):
+    """The PennyLane-Lightning plugin provides a fast state-vector simulator written in C++."""
 
-    homepage = "https://github.com/PennyLaneAI/pennylane-lightning"
-    url = "https://github.com/PennyLaneAI/pennylane-lightning/archive/refs/tags/v0.28.0.tar.gz"
-    # using github for now, because pypi tarball is missing the CMakeLists.txt file
-    # pypi = "PennyLane-Lightning/PennyLane-Lightning-0.28.0.tar.gz"
+    homepage = "https://docs.pennylane.ai/projects/lightning"
+    git = "https://github.com/PennyLaneAI/pennylane-lightning.git"
+    url = "https://github.com/PennyLaneAI/pennylane-lightning/archive/refs/tags/v0.28.2.tar.gz"
+    tag = "v0.28.2"
 
-    version("0.28.0", sha256="f5849c2affb5fb57aca20feb40ca829d171b07db2304fde0a37c2332c5b09e18")
+    maintainers = ["mlxd", "AmintorDusko"]
```

Comment:
```suggestion
    maintainers("mlxd", "AmintorDusko")
```
We have a new directive that appends instead of overwrites, so I will still be listed as a maintainer too.